installers/unix/env/install.sh

```diff
@@ -62,6 +61,13 @@ realpath() {
 IFS="
 "
 MACOS=$(echo $OSTYPE | grep darwin || true)
+if [ $MACOS ]; then
+    RC=$HOME/.profile;
```

Comment:
without ``'`` it will be actualize when ``RC`` is assigned.

Reply:
That's what I expect